PowerPointLabs/PowerPointLabs/ThisAddIn.cs

```diff
@@ -978,6 +979,21 @@ private void ThisAddInSelectionChanged(PowerPoint.Selection sel)
 
             }
 
+
+            // When there is no selection on the slide, disable the add/copy buttons on
+            // customshapepane and syncpane respectively
+            if (GetActivePane(typeof(SyncPane)) != null)
+            {
+                SyncPane syncPane = GetActivePane(typeof(SyncPane)).Control as SyncPane;
+                syncPane.UpdateOnSelectionChange();

```

Comment:
Why not pass the selection `sel` to this method as a parameter too?

Reply:
This is because the selection can be retrieved inside `SyncPaneWPF.xaml.cs` but not in `ShapesLab`. I can change it such that both methods take in `sel`!

Reply:
Thanks! Hopefully nothing goes wrong with that! Just felt it is slightly better to use `sel` instead of `this.GetCurrentSelection()` because we have received #1543  bug reports before about `this.GetCurrentSelection()` sometimes not working in certain environments (not sure why...)